src/services/BiometricAuth.js

```diff
@@ -117,7 +124,7 @@ export default class BiometricAuth extends React.PureComponent {
 								icon={['fa', 'fingerprint']}
 								size={75}
 								style={{
-									color: accepted ? colors.primary : colors.secondary,
+									color: colors.secondary,
```

Comment:
So no changes if it is correctly detected ?

Reply:
The modal is closed when it it's ok